manifests/params.pp

```diff
@@ -43,10 +43,10 @@
       timeout  =>  5,
     }
   ]
-  $host            =     '0.0.0.0'
+  $host            =     '127.0.0.1'
   $port            =     3000
   $user            =     ''
-  $pass            =     ''
+  $pass            =     hiera('default_pass')
```

Comment:
Don't bother with this direct hiera lookup. Just put 'needs_to_be_changed' if that is what you want to be the default.